src/Features/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -38,12 +38,14 @@ static Test()
                 // reasonable TLS protocol version for outgoing connections.
 #pragma warning disable CA5364 // Do Not Use Deprecated Security Protocols
 #pragma warning disable CS0618 // Type or member is obsolete
+#pragma warning disable SYSLIB0014 // 'ServicePointManager' is obsolete
```

Comment:
Is this obsoleted in a way that means this isn't working anymore? Is this just stale now?

Reply:
(I have no idea what this is, but might be worth filing a bug as a follow up to track what this is.)

Reply:
WebRequest, WebClient, and ServicePoint are obsolete since .NET 6. See https://learn.microsoft.com/en-us/dotnet/core/compatibility/networking/6.0/webrequest-deprecated. I'm not sure why the SYSLIB0014 warning wasn't reported previously, will investigate that.